lmp/bb-config.sh

```diff
@@ -36,6 +37,12 @@ fi
 
 source setup-environment build
 
+if [ "$DEV_MODE" == "1" ]; then
+	cat << EOFEOF >> conf/local.conf
```

Comment:
LGTM.

PS. I'm just curious, why `cat` in-place doc and not `echo`? Like:
```
echo 'DEV_MODE = "1"' >> conf/local.conf
```

Reply:
I only copied what is already there. I remember I saw this discussion somewhere, some time ago, but I really don't remember. @MrCry0

Reply:
Yeah, in this case both would work.

I believe we used cat in order to support multiple lines more easily.